packages/app-mobile/package.json

```diff
@@ -50,7 +50,7 @@
     "react-native-image-picker": "^2.3.4",
     "react-native-image-resizer": "^1.3.0",
     "react-native-modal-datetime-picker": "^9.0.0",
-    "react-native-popup-menu": "^0.10.0",
+    "react-native-popup-menu": "^0.15.13",
```

Comment:
Updated to fix missing type information. (Versions `0.12.5+` have TypeScript definitions).